### DISCUSSION TITLE
Set Database Dialect for Connections in serenity.net 5

### DISCUSSION THREAD

**Discussion by dfaruque** [serenity-is/Serenity#6089] [Help] [answered]
What is the alternative way of doing the following in serenity.net 5?
 in older version we did
`SqlConnections.GetConnectionString("Default").Dialect = Oracle12cDialect.Instance;`

ref. doc: https://serenity.is/docs/setting-connection-dialect

Thanks

**Comment by VictorTomaili** [comment, marked as answer]
if provider is not present in the `DefaultConnectionStrings`, you can set default by `Serenity.Data.SqlSettings.DefaultDialect = somedialect;` 
Automatic resolve by provider https://github.com/serenity-is/Serenity/blob/master/src/Serenity.Net.Data/Connections/DefaultConnectionStrings.cs#L25
For custom requirements, create a customized `IConnectionString` class
